app/src/main/res/values-az/strings.xml

```diff
@@ -31,6 +31,7 @@
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
     <string name="bottomBar_bookmarks">Əlfəcinlərimə keç</string>
+    <string name="app_bars_dot_spacer">·</string>
```

Comment:
No, i mean remove it here but modify the strings.xml in values to add the translatable="false"